### ISSUE TITLE
Bug: Plex Stream 

### ISSUE THREAD

**Issue by lavazza2017** [SenexCrenshaw/StreamMaster#144]
**Describe the Bug**
Hi, I have imported my m3u link and epg list (this is working fine if I don't start play with another epg list), then I added new STREAM GROUP called "test" and copy link in to Plex. 
Icon, epg, programs import fine but when I have tried run any of programs it fails.
When I downloaded M3U from SM and play on any player it working fine.
When I copy link from SM in play list it is working too. 

problem is with transferring information to plex 

**Stream Master Info**
lates ->0.3.0.0

**Link to logs (required)**

**Comment by jaragoncr99** [comment]
It's weird because this happens intermittently, right now its working just fine for me.

**Comment by lavazza2017** [comment]
yes, you are 100% right, I set 3-4 times image in docker and 2 times just fail (same steps, same m3u and epg). Unfortunately when you add another epg list program is starting be very unstable.
In addition is there way in SM to download from EPG automatically icon ? (I found option "Cache Icons" and however plex never see that icons)

**Comment by SenexCrenshaw** [comment]
> yes, you are 100% right, I set 3-4 times image in docker and 2 times just fail (same steps, same m3u and epg). Unfortunately when you add another epg list program is starting be very unstable. In addition is there way in SM to download from EPG automatically icon ? (I found option "Cache Icons" and however plex never see that icons)

different bug. open a new one if needed

**Comment by SenexCrenshaw** [comment]
plex will get buggy if you add/remove a HDHR with the same deviceid. Try changing that in them SM UI settings and see if that helps.

**Comment by SenexCrenshaw** [comment]
another test is past an m3u link from SM into vlc and see if that works. if it does, change the device id and redo it in plex.

**Comment by evulhotdog** [comment]
This may or may not be related to this issue, however in my experience Plex has a bunch of very poor implementations for watching and recording IPTV. I'd recommend basically any other client, or even Emby or Jellyfin, over Plex.

**Comment by SenexCrenshaw** [comment]
> This may or may not be related to this issue, however in my experience Plex has a bunch of very poor implementations for watching and recording IPTV. I'd recommend basically any other client, or even Emby or Jellyfin, over Plex.

its questionable at best!
